logstash-verifier/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM msheiny/debian-jessie-systemd:latest
```

Comment:
Can I get the `msheiny/debian-jessie-systemd` image in here too? I'll handle converting and standardizing on the FPF Quay registry, while adding the timestamp tags.

Reply:
Will handle msheiny -> freedomofpress for systemd container as a separate issue.

Reply:
oh whoops -- i didnt realize that was tagged in the dockerfile here, should be an easy switch-over to our FPF quay hosted one